File: 2 semestre/Laboratório de Desenvolvimento de Algoritmos/UNICID - Semestre 2/src/aula10_01_11_16Vetores/SomaDiferenca.java
//Importando Scanner
import java.util.Scanner;

public class SomaDiferenca {

	public static void main(String[] args){
		
		//Instanciando Scanner
		Scanner input = new Scanner(System.in);
		
		System.out.println("Soma e Diferença de Vetores\n");
		
		//vars
		int a[] = new int[6];
		int b[] = new int[6];
		int soma[] = new int[6];
		int dif[] = new int[6];
		int i;
		
		//Entradas
		//a
		for( i=0; i<6; i++)
		{
			System.out.print("Digite A[" + i+ "]: ");
			a[i] = input.nextInt();
		}
		
		//b
		for( i=0; i<6; i++)
		{
			System.out.print("Digite B[" + i + "]: ");
			b[i] = input.nextInt();
		}
		
		//Processamento
		for( i=0; i<6; i++)
		{
			soma[i] = a[i] + b[i];
			dif[i] = a[i] - b[i];
		}
		
		//Saída
		System.out.println("\nVetores:");
		
		//A
		System.out.println("\nA:");
		for(i=0; i<6; i++)
		{
			System.out.print(a[i]+", ");
		}
		
		//B
		System.out.println("\nB:");
		for(i=0; i<6; i++)
		{
			System.out.print(b[i]+", ");
		}
		
		//Soma
		System.out.println("\nSoma:");
		for(i=0; i<6; i++)
		{
			System.out.print(soma[i]+", ");
		}
		
		//Diferença
		System.out.println("\nDiferença:");
		for(i=0; i<6; i++)
		{
			System.out.print(dif[i]+", ");
		}
		
		
	}
	
}

File: 2 semestre/Técnica de Desenvolvimento de Algoritmos/Aula 12 03-11-16/somaDiferenca.c
#include <stdio.h>

main()
{
	printf("Soma e Diferenca de Vetores\n\n");
	
	//vars
	int a[6], b[6], soma[6], dif[6], i;
	
	//Entradas
	for( i=0; i<6; i++ )
	{
		printf("Digite a[%i]: ", i);
		scanf("%d",&a[i]);
	}
	
	for( i=0; i<6; i++ )
	{
		printf("Digite b[%d]: ", i);
		scanf("%i",&b[i]);
	}
	
	//Processamento
	for( i=0; i<6; i++)
	{
		soma[i] = a[i] + b[i];
		dif[i] = a[i] - b[i];
	}
	
	//Saída
	printf("\n\nVetor Soma: \n");
	for( i=0; i<6; i++)
	{
		printf("%d  ", soma[i]);
	}
	
	printf("\nVetor Diferença: \n");
	for( i=0; i<6; i++ )
	{
		printf("%d  ", dif[i]);
	}
}
